findEmbeddedContent symbol missing

After installing the latest version of seasocks, I'm having trouble building against `libseasocks.a`:

```
/usr/local/lib/libseasocks.a(Connection.cpp.o): In function `seasocks::Connection::sendError(seasocks::ResponseCode, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)':
Connection.cpp:(.text+0x4c2c): undefined reference to `findEmbeddedContent(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/local/lib/libseasocks.a(Connection.cpp.o): In function `seasocks::Connection::send404()':
Connection.cpp:(.text+0x52b5): undefined reference to `findEmbeddedContent(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/local/lib/libseasocks.a(Connection.cpp.o): In function `seasocks::Connection::processHeaders(unsigned char*, unsigned char*)':
Connection.cpp:(.text+0x6424): undefined reference to `findEmbeddedContent(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
collect2: error: ld returned 1 exit status
```

--- a/src/main/c/CMakeLists.txt
+++ b/src/main/c/CMakeLists.txt
@@ -23,13 +23,13 @@ add_library(seasocks_obj OBJECT ${SEASOCKS_SOURCE_FILES})
 target_include_directories(seasocks_obj PUBLIC .)
 set_property(TARGET seasocks_obj PROPERTY POSITION_INDEPENDENT_CODE TRUE)
 
-add_library(seasocks STATIC $<TARGET_OBJECTS:seasocks_obj>)
+add_library(seasocks STATIC $<TARGET_OBJECTS:seasocks_obj> $<TARGET_OBJECTS:embedded>)
 target_include_directories(seasocks PUBLIC .)
-target_link_libraries(seasocks PRIVATE ${CMAKE_THREAD_LIBS_INIT} embedded)
+target_link_libraries(seasocks PRIVATE ${CMAKE_THREAD_LIBS_INIT})
 
-add_library(seasocks_so SHARED $<TARGET_OBJECTS:seasocks_obj>)
+add_library(seasocks_so SHARED $<TARGET_OBJECTS:seasocks_obj> $<TARGET_OBJECTS:embedded>)
 target_include_directories(seasocks_so PUBLIC ${ZLIB_INCLUDE_DIRS} .)
-target_link_libraries(seasocks_so PRIVATE ${CMAKE_THREAD_LIBS_INIT} embedded "${ZLIB_LIBRARIES}")
+target_link_libraries(seasocks_so PRIVATE ${CMAKE_THREAD_LIBS_INIT} "${ZLIB_LIBRARIES}")
 set_target_properties(seasocks_so PROPERTIES OUTPUT_NAME seasocks)
 
 install(TARGETS seasocks seasocks_so

--- a/src/main/web/CMakeLists.txt
+++ b/src/main/web/CMakeLists.txt
@@ -16,6 +16,6 @@ add_custom_command(OUTPUT Embedded.cpp
                         COMMAND ${PYTHON_BIN} ${SCRIPT} ${EMBEDDED_FILES} $<ANGLE-R> Embedded.cpp
                         COMMENT "Generating embedded content"
                         )
-add_library(embedded STATIC Embedded.cpp)
+add_library(embedded OBJECT Embedded.cpp)
 target_include_directories(embedded PUBLIC $<TARGET_PROPERTY:seasocks,INTERFACE_INCLUDE_DIRECTORIES>)
 set_property(TARGET embedded PROPERTY POSITION_INDEPENDENT_CODE TRUE)

